habitat-lab/habitat/tasks/rearrange/articulated_agent_manager.py

```diff
@@ -77,6 +77,8 @@ def __init__(self, cfg, sim):
             agent_cfg = cfg.agents[agent_name]
             agent_cls = eval(agent_cfg.articulated_agent_type)
             agent = agent_cls(agent_cfg.articulated_agent_urdf, sim)
+            if agent_cfg.motion_data_path != "":
```

Comment:
I prefer not having this added complexity here and keeping the motion_data_path internal to the humanoid components.